### PR TITLE
pushed code to integrate admin registration form to the admin tab

### PR DESCRIPTION
**Pull request by aaryaamoharir**

### CODE REVIEW

File: app/styles/volunteerAdmin.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* header.css */
```
Comment (devAgant) [inline app/styles/volunteerAdmin.css:1]: Please use TailwindCSS and not regular CSS files. Refer to https://tailwindcss.com/

File: app/dashboardEmployee/admin/page.jsx [hunk 2]
```diff
@@ -1,4 +1,17 @@
+
+import Image from 'next/image';
+import "/app/styles/volunteerAdmin.css"
+
 export default function Page() {
-  return <p>Admin Page</p>;
+  return (
+    <div className="flex items-center justify-center h-[80vh] bg-[#fbfcf8] w-full" >
+      <div className="page">
+      <h1 class="text-center text-[#333333]">Welcome to the Admin Portal</h1>
```
Comment (devAgant) [inline app/dashboardEmployee/admin/page.jsx:9]: className

File: app/dashboardEmployee/admin/page.jsx [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+        <a href="/registration">
+        <button class="flex items-center justify-center text-center w-1/2 h-[50px] p-2 bg-[#419902] text-white font-bold text-lg cursor-pointer my-4 mx-auto transition-colors duration-300 hover:bg-[#357a01]">
```
Comment (devAgant) [inline app/dashboardEmployee/admin/page.jsx:11]: className

**Review by devAgant — CHANGES_REQUESTED** [review]

**Review by CalvinVan — COMMENTED** [review]
It might not be necessary anymore to have the volunteerAdmin.css since we've converted to tailwind so you can probably delete it.

**Review by CalvinVan — DISMISSED** [review]
On line 3 I believe?, I can't quote the line directly. You need to remove the line that is trying to import your css file which doesn't exist anymore.

**Review by devAgant — APPROVED** [review]

**Pull request merged**